pkg/nuctl/command/parse.go

```diff
@@ -80,6 +80,11 @@ func (pc *parseCommandeer) ParseReport(ctx context.Context, logger logger.Logger
 		if err = json.Unmarshal(reportData, kind); err == nil {
 			t := table.NewWriter()
 			kind.PrintAsTable(t, pc.onlyFailed)
+
+			// do not print a report if it is empty
+			if t.Length() == 0 {
```

Comment:
Note that `PrintAsTable` always adds the table headers:
```
t.AppendHeader(tableHeader)
```

Will the length still be 0 even with headers?
Unit tests will tell for sure 😉

Reply:
@TomerShor I'm uncertain if testing the library is necessary, but I've included a simple test to demonstrate that the length is zero despite the presence of headers.